sound/soc/intel/boards/glk_rt5682_max98357a.c

```diff
@@ -409,6 +409,7 @@ static struct snd_soc_dai_link geminilake_dais[] = {
 		.init = NULL,
 		.capture_only = 1,
 		.nonatomic = 1,
+		.dynamic = 1,
```

Comment:
The commit message is a bit misleading. It's more like this is required for SOF to ignore hard-coded front-ends signalled with the .dynamic field